src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck started working");

```

Comment:
The message 'The truck started working' should be more informative. It's better to indicate that it is a Truck that started working. For example: 'Truck is doing its work'.